src/decorators.ts

```diff
@@ -38,13 +39,48 @@ export function decorators() {
         const { document } = editor
         const source = document.getText()
 
+        function createIndexDecorator(position: vscode.Position, index: number) {
+          const decorator = createDecorator(`i:${index}`)
+          decorators.push(decorator)
+          editor.setDecorations(decorator, [new vscode.Range(position, position)])
+        }
+
+        function markArrayLiteralsInGolang(document: vscode.TextDocument) {
+          const filePath = document.fileName.substring(0, document.fileName.length - 3)
+          
+          const goToolsGopath = vscode.workspace.getConfiguration('go')['toolsGopath']??""
+          // FIXME: path to real main.go
+          const parserScriptPath = path.join(__dirname, "..","src", 'go', 'parser', 'main.go')
+
+          let goBinPath = "go" // Maybe check ext fot windows?
+          if(goToolsGopath !== "") {
+            goBinPath = `${goToolsGopath}/`+goBinPath
```

Comment:
Here one may first try
```ts
child_process.spawn(goBinPath, ["version"]);
```
if it doesn't work, then try it with `goBinPath = "go"` before ultimately failing.

Anyway, parser is more or less toolchain-agnostic, so it can be even cached across multiple `go.toolsGopath`s

Reply:
Though better cache with go.toolsGoPath, because the `go` versions might differ